assets/training/model_management/components/validation_trigger_import/spec.yaml

```diff
@@ -0,0 +1,178 @@
+      eg: trust_remote_code=True, device_map=auto
+    optional: true
+
+  hf_config_class:
+    type: string
+    description: AutoConfig class may not be sufficient to load config for some of the models. You can use this parameter to send Config class name as it is
+    optional: true
+
+  hf_model_class:
+    type: string
+    description: AutoModel classes may not be sufficient to load some of the models. You can use this parameter to send Model class name as it is
+    optional: true
+
+  hf_tokenizer_class:
+    type: string
+    description: AutoTokenizer class may not be sufficient to load tokenizer for some of the models. You can use this parameter to send Config class name as it is
+    optional: true
+
+  hf_use_experimental_features:
+    type: boolean
+    description: Enable experimental features for hugging face MLflow model conversion
+    default: false
+    optional: true
+
+  extra_pip_requirements:
+    type: string
+    description: |
+      Extra pip dependencies that MLflow model should capture as part of conversion. This would be used to create environment while loading the model for inference.
+      Pip dependencies expressed as below. Do not use quotes for passing.
+      eg: pkg1==1.0, pkg2, pkg3==1.0
+    optional: true
+
+  ## Inputs for MLflow local validation
+  local_validation_test_data:
+    type: uri_file
+    optional: true
+    description: Test data for MLflow local validation. Validation will be skipped if test data is not provided
+
+  local_validation_column_rename_map:
+    type: string
+    optional: true
+    description: |
+      Provide mapping for local validation test data column names, that should be renamed before inferencing
+      eg: col1:ren1; col2:ren2; col3:ren3
+
+  ## Inputs for Model registration
+  custom_model_name:
+    type: string
+    optional: true
+    description: Model name to use in the registration. If name already exists, the version will be auto incremented
+
+  model_version:
+    type: string
+    optional: true
+    description: Model version in workspace/registry. If the same model name and version exists, the version will be auto incremented
+
+  model_description:
+    type: string
+    optional: true
+    description: Description of the model that will be shown in AzureML registry or workspace
+
+  registry_name:
+    type: string
+    optional: true
+    description: Name of the AzureML asset registry where the model will be registered. Model will be registered in a workspace if this is unspecified
+
+  model_metadata:
+    type: uri_file
+    optional: true
+    description: A JSON or a YAML file that contains model metadata confirming to Model V2 [contract](https://azuremlschemas.azureedge.net/latest/model.schema.json)
+
+  update_existing_model:
+    type: boolean
+    default: false
+    description: If set to true, will update the existing model. If set to false, will create a new model.
```

Comment:
nit: elaborate description what `update the existing model` means and `where new model will be created`. basically src and dstn is different, and we can add that information in description.

Reply:
Ack